cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -82,4 +93,36 @@ <h4 class="u-mb5">
     </li>
     {% endfor %}
 </ul>
+
+{% if settings.view_more %}
+  {{ _view_more(settings.view_more) }}
+{% endif %}
+
+{% endmacro %}
+
+{# ==========================================================================
+
+   _view_more()
+
+   ==========================================================================
+
+   Description:
+
+   Creates view more markup when given:
+
+   settings:      An object used to customize the markup.
```

Comment:
We've been using `link_url` and `link_text` so might as well stick with that for consistency.